### PR TITLE
Put added in custom request.

### LINKED ISSUE #185: PUT method in CustomResquest class
Hi,

I'm trying to use CustomRequest to create a team from an already existing group (an education class, really), as there is still no teams implementation in the SDK.

The problem is that the CustomRequest class does not provide a put method. And setting the httpMethod in the GraphClient is useless, as far as I'm concerned.

I have modified the CustomRequest class including the put method, and it works fine, but it would be great if it where included in the project.

### Expected behavior

CustomRequest<JsonObject> teamRequest = graphClient.customRequest("/groups/" + educationClass.id + "/team").buildRequest();			
JsonObject jsonRequestBody...
JsonObject result = teamRequest.**put**(jsonRequestBody);

### Actual behavior

CustomRequest<JsonObject> teamRequest = graphClient.customRequest("/groups/" + educationClass.id + "/team").buildRequest();			
JsonObject jsonRequestBody...
JsonObject result = teamRequest.???(jsonRequestBody); --> No PUT method available

### PR DESCRIPTION
**Pull request by NakulSabharwal**
Fixes #185 

### Changes proposed in this pull request
- Put functionality added in Custom request.

### CODE REVIEW

File: src/test/java/com/microsoft/graph/functional/CustomRequestTests.java [hunk 1]
```diff
@@ -41,4 +43,22 @@ public void testCustomGet() {
 		assertEquals(meGraphService.displayName, meOriginal.displayName);
 		assertEquals(meJson.get("displayName").getAsString(), meOriginal.displayName);
 	}
+	
+	/**
+	 * Test PUT with a custom request for both serialized and JSON content
+	 */
```
Comment (deepak2016) [inline src/test/java/com/microsoft/graph/functional/CustomRequestTests.java]: you added two put methods, but only one test, can you please add one more test covering other function?

Comment (NakulSabharwal) [reply src/test/java/com/microsoft/graph/functional/CustomRequestTests.java]: Updated tests to cover both Put functions added.

Comment (deepak2016) [reply src/test/java/com/microsoft/graph/functional/CustomRequestTests.java]: hey, ideally you should create two separate tests for two methods.

**Comment by darrelmiller** [comment]
I'd like to discuss further why we actually have a customRequest class.  When we move to the Apache HttpClient library that has a native request object, will this be needed anymore?  The overall idea is that we can convert the output of a Request Builder to a native request object , so that users can do what ever manipulation you want.  

Adding PUT to the customRequest class feels like we are slowly re-inventing a native HTTP library.

**Comment by deepak2016** [comment]
As discussed, we are going ahead and merging this pull request.

**Pull request merged**